toolkits/global/packages/global-forms/README.md

```diff
@@ -80,10 +75,10 @@ The `hint` property adds hint text under the main label text but _inside_ the `<
 "hint": "This will be the email address you used when registering"
 ```
 
-The `optional` property adds _“(optional)”_ to the label text.
+To accompany the `required` property, there is `requiredSuffix`. You can use this to add an indicator after the label text, such as an asterisk. HTML is permitted, meaning you can hide the asterisk from assistive technologies using `aria-hidden`. The presence of the `required` attribute on the input itself suffices for screen reader software.
 
 ```json
-"optional": true
+"requiredSuffix": "<span aria-hidden=\"true\">*</span>"
```

Comment:
See previous comment. 

I also think it might be good to add some guidance around this, e.g.:

Avoid asking for information you don't really need. If you do need to ask for optional information, set the `optional` property to `true`.

Reply:
@amyhupe See above. Please check the rewording of that section.